src/ucs/config/global_opts.c

```diff
@@ -311,3 +312,41 @@ void ucs_global_opts_print(FILE *stream, ucs_config_print_flags_t print_flags)
                                  ucs_global_opts_table, NULL,
                                  UCS_DEFAULT_ENV_PREFIX, print_flags);
 }
+
+static void ucs_vfs_read_log_level(void *obj, ucs_string_buffer_t *strb,
+                                   void *arg_ptr, uint64_t arg_u64)
+{
+    ucs_log_level_t ucs_log_level = ucs_global_opts.log_component.log_level;
+    ucs_string_buffer_appendf(strb, "%s\n", ucs_log_level_names[ucs_log_level]);
+}
+
+static ucs_status_t ucs_vfs_write_log_level(void *obj, const char *buffer,
+                                            size_t size, void *arg_ptr,
+                                            uint64_t arg_u64)
+{
+    unsigned log_level;
+    ucs_string_buffer_t strb;
+
+    ucs_string_buffer_init(&strb);
+    ucs_string_buffer_appendf(&strb, "%s", buffer);
+    ucs_string_buffer_rtrim(&strb, "\n");
```

Comment:
why need to trim?

Reply:
Because buffer is passed from write file system operation with '\n' in the end.